website/thaliawebsite/management/commands/migratemedia.py

```diff
@@ -0,0 +1,45 @@
+from django.core.files.base import ContentFile
+from django.core.management.base import BaseCommand
+from django.conf import settings
+
+import logging
+import os
+import boto3
+from botocore.exceptions import ClientError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        # create session to s3
+        session = boto3.session.Session()
+        s3_client = session.client(
+            service_name="s3",
+            aws_access_key_id=settings.AWS_ACCESS_KEY_ID,
+            aws_secret_access_key=settings.AWS_SECRET_ACCESS_KEY,
+            endpoint_url=f"https://{settings.AWS_STORAGE_BUCKET_NAME}.s3.eu-west-1.amazonaws.com/",
```

Comment:
this endpoint should also be in settings I think

Reply:
It is in settings, but as PUBLIC_MEDIA_URL. And after further looking into it I don't think we really need it since boto3 can construct this based on the bucket name. And I just copied this from an example :)

Reply:
So do we want to change this?